src/mappings/core.ts

```diff
@@ -57,7 +57,7 @@ export function handleTransfer(event: Transfer): void {
 
   // get or create transaction
   let transaction = Transaction.load(transactionHash)
-  if (transaction == null) {
+  if (transaction === null) {
```

Comment:
Does AssemblyScript support `===`? I vaguely remember trying it _ages ago_ and it failed to compile even.

Reply:
Yes it does! It's a bit weird though which is why it even get an honorary mention in the "quirks" section :-D https://www.assemblyscript.org/basics.html#quirks

Reply:
Btw. I highly recommend that website / docs. They are written very well and because of the way they are structured (not just like a technical spec but literally like a guidebook for navigating assemblyscript from the perspective of a developer that is farmiliar with javascript) they are super helpful. Maybe you could add a link to that website from the thegraph docs? At least people on my team often struggle with the quirks and differences between js/ts/assemblyscript simply because "it's so similar but not quite the same".